lib/locreator.js

```diff
@@ -12,6 +13,15 @@ var Locreator = function (nemo) {
   this.drivex = Drivex(nemo.driver, nemo.wd);
   this.locatex = Locatex(nemo);
   this.normify = this.normalize = function (locator) {
+    if(locator.constructor !== String) {
+      if(!locator.locator || !locator.locator.trim()) {
+        throw new Error('NO or EMPTY locator found, please fix it');
+      }
+      if(!locator.type || !locator.type.trim() || !nemo.wd.By[locator.type]) {
```

Comment:
That's definitely a gray area. The way I would describe it is, the Object definition _requires_ type. If someone wants to use the "css by default" behavior, they can structure their view JSON to use the string locators instead of the Object locators. If someone is attempting to use an Object locator, we require a valid "type". If they aren't including a type, their locator would look like this:

```
"myLocator": {
  "locator": ".foo .bar #bing"
}
```

In that case, the JSON structure is just extraneous and would be better written as:

```
"myLocator": ".foo .bar #bing"
```